games-emulation/dolphin-libretro/dolphin-libretro-1.0_pre20250616.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LIBRETRO_REPO_NAME="libretro/dolphin"
+LIBRETRO_COMMIT_SHA="a09f78f735f0d2184f64ba5b134abe98ee99c65f"
+LIBRETRO_CORE_NAME="dolphin"
+
+inherit libretro-core cmake
+
+DESCRIPTION="A Gamecube/Wii emulator core for libretro"
+HOMEPAGE="https://github.com/libretro/dolphin"
+KEYWORDS="~amd64 ~arm64"
+IUSE="+opengl vulkan +X"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+DEPEND="
+	<dev-libs/libfmt-10
```

Comment:
Compilation breaks with libfmt-11.0.2 that is currently stable. The error message is rather cryptic, might log a bug on dolphin-libretro, see if upstream sorts this out.

Reply:
It was trying to use an undefined type. My impression is the older libfmt pulls in some bloat and that makes it work. However the error message was Teutonic, so quite couldn't spot it.